src/app/(proper_react)/layout.tsx

```diff
@@ -28,6 +29,9 @@ export default async function Layout({ children }: { children: ReactNode }) {
     <L10nProvider bundleSources={l10nBundles}>
       <ReactAriaI18nProvider locale={getLocale(l10nBundles)}>
         <CountryCodeProvider countryCode={countryCode}>
+          {enabledFlags.includes("PromptNoneAuthFlow") && !session && (
```

Comment:
Maybe a bit of a nit, but to decrease the amount of client code that's downloaded, parsed, and run by everyone, would it make sense to check the query parameters here to only conditionally include the component, and then in the component initiate the sign-in flow whenever it is rendered?

Reply:
I like that idea. Unfortunately, Next.js does not support accessing query parameters in `layout` components.

Reply:
Ah right, nah, client-side is fine then.